lib/src/sliverscrollviewcalendar.dart

```diff
@@ -447,12 +447,15 @@ class SliverScrollViewCalendarElement extends StatelessElement
     SliverScrollViewCalendar calendarWidget = widget;
     return new SliverListCenter(
       startIndex: _startDisplayIndex,
-         state: _state,
-         controller: calendarWidget.controller,
+      state: _state,
+      controller: calendarWidget.controller,
       delegate: new SliverChildBuilderDelegate(
         (BuildContext context, int index) {
+          if (index < _beginningRangeIndex || (_endingRangeIndex != -1 && index > _endingRangeIndex)) {
```

Comment:
I think it is better to limit scrolling, also suggested by https://github.com/pinkfish/flutter_calendar/issues/9

Reply:
Makes sense.  I need infinite scroll so didn't try the clipping.  The sliver stuff was a little fiddly to get working correctly.  Looks like you tested it though.  I think the only thing I would prefer to fix is the timezone to use the dart package directly rather than github.  Was not a big fan of the github link really.